manuscript/02-bestandteile-eines-modellgetriebenen-designs.md

```diff
@@ -1,177 +1,180 @@
-# II Bestandteil eines modellgetriebenen Designs
+# II. Bestandteile eines modellgetriebenen Entwurfs
 
-Diese Pattern stellen weit verbreitete Best Practices für
-objekt-orientiertes Designs im Hinblick auf domain-driven Design dar.
-Sie leiten die Entscheidungen zur Präzisierung des Modells und halten
-Modell und Implementierung aufeinander abgestimmt, wobei das eine die
-Effektivität des anderen verstärkt.  Das sorgfältige Ausarbeiten der
+Diese Muster stellen weit verbreitete bewährte Praktiken für
+objekt-orientiertes Designs im Hinblick auf Domain-Driven Design dar.
+Sie leiten Entscheidungen, welche das Modell präzisieren und das Modell 
+und die Implementierung aufeinander abgestimmt halten, wobei das eine 
+die Effektivität des anderen verstärkt. Das sorgfältige Ausarbeiten der
 Details der einzelnen Modellelemente gibt den Entwicklern eine solide
 Plattform, um Modelle zu erforschen und in enger Abstimmung mit der
 Implementierung zu halten.
 
-## Layered Architecture {#layered-architectur}
+## Layered Architecture {#layered-architecture}
 
-*Dt.: Geschichte Architektur*
+*dt.: Geschichtete Architektur*
 
-In einem objektorientierten Programm werden Benutzeroberfläche,
+In einem objekt-orientierten Programm werden Benutzeroberfläche,
 Datenbank und anderer unterstützender Code oft direkt in den
-Business-Objekte implementiert.  Zusätzliche Geschäftslogik ist in das
-Verhalten von UI-Widgets und Datenbankskripten eingebettet.  Dies
+Business-Objekte implementiert. Zusätzliche Geschäftslogik ist in das
+Verhalten von UI-Widgets und in Datenbankskripten eingebettet. Dies
 geschieht, weil es der einfachste Weg ist, die Dinge schnell zum
-Laufen zu bringen.
-
-Wenn der domänenbezogene Code durch eine so große Menge an anderem
-Code diffus wird, wird es extrem schwierig, ihn zu identifizieren und
-zu verstehen.  Oberflächliche Änderungen an der Benutzeroberfläche
-können in Wirklichkeit die Geschäftslogik verändern.  Um eine
-Geschäftsregel zu ändern, kann das sorgfältige Prüfung von UI-Code,
-Datenbankcode oder anderen Programmelementen erforderlich sein.  Die
+Laufen zu bringen, allerdings nur auf kurze Dauer.
+
+Wenn der domänenbezogene Code durch eine so große Menge an andersartigem
+Code verwischt wird, wird es extrem schwierig, ihn zu identifizieren und
+zu verstehen. Einfache Änderungen an der Benutzeroberfläche
+können in Wirklichkeit die Geschäftslogik verändern. Um eine
+Geschäftsregel zu ändern, kann das akribische Prüfen von UI-Code,
+Datenbankcode oder anderen Programmteilen erforderlich sein. Die
 Implementierung kohärenter, modellgetriebener Objekte wird
-unpraktikabel.  Automatisiertes Testen ist umständlich.  Mit all den
+unpraktikabel. Automatisiertes Testen ist umständlich. Mit all den
 Technologien und der Logik, die in jede einzelne Aktivität involviert
-sind, muss ein Programm muss sehr einfach gehalten werden, oder es
+ist, muss ein Programm sehr einfach gehalten werden, oder es
```

Comment:
IMHO bezieht sich der Satz ",die in jede..." auf Logik *und* Technologien. Dann wäre das Verb im Plural.

Reply:
Ja, kann man auch so interpretieren, und stimmt wohl eher. Fixed in 4850306985c2739f4f0f3624068c7a7c244764ae